### PR TITLE
Update gh-pages for 0.2

### LINKED ISSUE #663: [DOC] Sample CUDA dockerfile should be public or a simple static file
## Report incorrect documentation

**Location of incorrect documentation**
https://nvidia.github.io/spark-rapids/docs/get-started/getting-started.html#running-on-kubernetes

**Describe the problems or issues found in the documentation**
Sample CUDA dockerfile should be public or a simple static file

**Steps taken to verify documentation is incorrect**
List any steps you have taken:

**Suggested fix for documentation**
A static file or a public google drive link helps.

---

## Report needed documentation

**Report needed documentation**
A clear and concise description of what documentation you believe it is needed and why.

**Describe the documentation you'd like**

**Steps taken to search for needed documentation**
List any steps you have taken:

### LINKED ISSUE #664: [DOC] Spark on k8s script is missing spark.jars
## Report incorrect documentation

**Location of incorrect documentation**
https://nvidia.github.io/spark-rapids/docs/get-started/getting-started.html#running-on-kubernetes

**Describe the problems or issues found in the documentation**
In the k8s docs, it is not mentioned that users would have to put the rapids jars under Spark directly and without including spark.jars, the driver and the executors cannot get access to the Rapids jars. 

**Steps taken to verify documentation is incorrect**
List any steps you have taken:

**Suggested fix for documentation**
Either include the spark.jars pointing to rapids jars or explicitly document to put the jars under the Spark jars/ directory so that the dependencies are part of Spark libs.  

---

## Report needed documentation

**Report needed documentation**
A clear and concise description of what documentation you believe it is needed and why.

**Describe the documentation you'd like**

**Steps taken to search for needed documentation**
List any steps you have taken:

### LINKED ISSUE #665: [DOC] Incorrect cudf jar name compared to the one available in downloads
## Report incorrect documentation

**Location of incorrect documentation**
https://nvidia.github.io/spark-rapids/docs/get-started/getting-started.html#download-the-rapids-jars

**Describe the problems or issues found in the documentation**
It is mentioned that the sample version of jar for CUDA-10.1 is cudf-0.15-SNAPSHOT-cuda10-1.jar but the one available in the download section is cudf-0.14-SNAPSHOT-cuda10-1.jar.

**Steps taken to verify documentation is incorrect**
List any steps you have taken:

**Suggested fix for documentation**
Correct the proper name accordingly.

---

## Report needed documentation

**Report needed documentation**
A clear and concise description of what documentation you believe it is needed and why.

**Describe the documentation you'd like**

**Steps taken to search for needed documentation**
List any steps you have taken:

### PR DESCRIPTION
**Pull request by sameerz**
Updated docs for gh-pages for 0.2.  

resolves #663 , resolves #664 , resolves #665

### CODE REVIEW

**Review by revans2 — APPROVED** [review]

File: docs/dev/README.md [hunk 1]
```diff
@@ -219,7 +223,7 @@ options, e.g.:
 
 ## Debugging Tips
 An easy way to debug the plugin is to run in
-[Spark local mode](https://jaceklaskowski.gitbooks.io/mastering-apache-spark/local/spark-local.html).
+[Spark local mode](https://spark.apache.org/docs/latest/spark-standalone.html).
```
Comment (jlowe) [inline docs/dev/README.md:226]: This section is referring to running Spark in local mode, but we're changing the link to refer to Spark standalone mode which is not the same thing.

**Review by jlowe — COMMENTED** [review]
We really need to get to a point where this is just a merge of the `docs/` directory from the `main` branch after a release, not hot-patching the docs commit-by-commit on the gh-pages branch.

File: docs/dev/README.md (hunk 1 above)
Comment (sameerz) [reply docs/dev/README.md]: Updated link to http://spark.apache.org/docs/latest/submitting-applications.html#master-urls

**Review by jlowe — APPROVED** [review]
Please followup with a PR against branch-0.3 to port over any missing changes there, otherwise our github.io docs and regular markdown docs will be out-of-sync on those changes.

**Pull request merged**